desktop/release-notes.md

```diff
@@ -24,12 +24,98 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.14
+
+2022-11-10
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Promoted Virtualization framework to default hypervisor for macOS >= 12.5.
```

Comment:
It would be good to make clear whether existing users will see a change or only new installations.